lib/shopify/resources/customer.ex

```diff
@@ -63,4 +65,34 @@ defmodule Shopify.Customer do
 
   @doc false
   def search_url, do: @plural <> "/search.json"
+
+  @doc false
+  def invite_url(id), do: @plural <> "/#{id}/send_invite.json"
+
+  @doc false
+  def orders(session, id, params \\ %{}) do
```

Comment:
There is a discussion about how to handle nested resources in #30 
the current consent is to use nested modules like `Shopify.Customer.Order.all/3`.

The nested resources are currently a little all over the place, but now that shopify cleaned up their API reference site, it might get a little easier 🌮  Lets stick to the nested modules for everything newly added.

So in this particular case I suggest to create `Shopify.Customer.Order.all/3` and use it in this convenience function
```elixir
def orders(session, id, params \\ %{}), do: Shopify.Customer.Order.all(session, id, params)
```

Reply:
Cool, done ;)